napari/utils/misc.py

```diff
@@ -148,8 +148,9 @@ def is_iterable(arg, color=False, allow_none=False):
     is taken to not be iterable. If allow_none is True, `None` is considered iterable.
     """
     if (
-        (arg is None and not allow_none)
-        or isinstance(arg, str)
+        isinstance(arg, (str, Enum))
+        or arg is None
+        and not allow_none
         or np.isscalar(arg)
```

Comment:
Nit: this at least needs some parentheses as before. But given the increased complexity, maybe can we just break this into a couple of early return conditions? I think that helps in this particular case, so this effectively ends up like a switch statement.